connectivity/check/result.go

```diff
@@ -105,8 +124,24 @@ func (r Result) String() string {
 		ret += "-"
 		ret += r.HTTP.Status
 	}
+	if r.Metrics != nil {
+		ret += displayMetrics(r.Metrics)
+	}
 	if r.ExitCode >= 0 && r.ExitCode <= 255 {
 		ret += fmt.Sprintf("-exit(%d)", r.ExitCode)
 	}
 	return ret
 }
+
+func displayMetrics(metrics []MetricsResult) string {
+	var ret string
+	ret += "-Sources("
```

Comment:
nit: Let's call this `MetricSources(` or `Metrics(`. It's not clear from context otherwise that this refers to metrics.